### PR TITLE
feat: Add the auto-coded icon in the view-expense screen

### PR DESCRIPTION
**Pull request by Z3RO-O**
## Clickup
https://app.clickup.com/t/86cw9uytj

## Code Coverage
Please add code coverage here

## UI Preview
image

## Summary by CodeRabbit

- **New Features**
	- Introduced a sparkle icon next to the "Spend Date" label to visually indicate when the extracted data matches the spent date.
  
- **Style**
	- Added a new CSS class for the sparkle icon to enhance its presentation, including size and spacing adjustments.

### CODE REVIEW

File: src/app/fyle/view-expense/view-expense.page.scss [hunk 1]
```diff
@@ -193,4 +193,16 @@
     margin-top: 16px;
     border-bottom: 1px solid $grey;
   }
+
+  &--auto-coded {
+    display: flex;
```
Comment (suyashpatil78) [inline src/app/fyle/view-expense/view-expense.page.scss:198]: Use `d-flex` class directly in template which is a globally defined class

**Review by suyashpatil78 — APPROVED** [review]
Few comments, please to fix them

File: src/app/fyle/view-expense/view-expense.page.scss (hunk 1 above)
Comment (Z3RO-O) [reply src/app/fyle/view-expense/view-expense.page.scss:198]: okay sir I will use this one

**Review by Dimple16 — APPROVED** [review]
In the screenshot, there's a tooltip - we agreed to not have one. Please verify this, rest lgtm

**Review by Chethan-Fyle — APPROVED** [review]

**Pull request merged**